### DISCUSSION TITLE
Vertical split

### DISCUSSION THREAD

**Discussion by Revell1on** [openscopeproject/TrguiNG#42] [Ideas]
Right now im using TrguiNG on macbook screen, and due to its size horizontal split (top are torrents, bottom are details) is not that convenient, please add an option and/or a setting to do a vertical split (left are torrents, right are details)

**Comment by qu1ck** [comment]
It's easy enough to make vertical split but it's a pain to reformat the details tab to fit into narrow vertical slot. And you will likely have horizontal scrolls in files table. Generally horizontal scrolls are much less convenient than vertical ones.

I'm curious how you use the program because I never have enough horizontal space to display the torrents table and have to live with horizontal scroll. I would never want to have even less horizontal space. Do you have a lot of columns disabled and narrowed?

**Reply by Revell1on** [reply]
yes, i have a lot of columns hidden and narrowed (also i got somewhat used to the default transmission interface with the vertical split, [example](https://transmissionbt.com/assets/images/homepage/carousel_homepage/screenshot_windows.png), this had all info i need packed in 1 line w/o tabs, but it was overall clunky and less functional and less clean and tidy than urs), as long as i use transmission on my router, i don't care that much about seed/peer ratio and etc, it's just put a .torrent to watchdir and wait some time to download and twice the ratio to upload
p.s. can you also please add "expand all" option when right clicked on some specific folder in the "files" tab ?
<img width="1727" alt="Screenshot 2023-07-09 at 21 50 19" src="https://github.com/openscopeproject/TrguiNG/assets/13023196/bf3dce9a-8c16-4d5f-83c8-e68e2836d5b4">

**Reply by qu1ck** [reply]
Implemented both, will be in next release.

**Reply by qu1ck** [reply]
This is now in [v0.7.0](https://github.com/openscopeproject/TrguiNG/releases/tag/v0.7.0)

**Reply by Revell1on** [reply]
u're the best, works as charm
